plugins/generic/dublinCoreMeta/DublinCoreMetaPlugin.php

```diff
@@ -181,110 +190,129 @@ public function monographFileView($hookName, $args)
     {
         $request = Application::get()->getRequest();
         $monograph = $args[1];
-        $publication = $monograph->getCurrentPublication();
         $publicationFormat = $args[2];
         $submissionFile = $args[3];
+        $requestArgs = $request->getRequestedArgs();
         $press = $request->getContext();
 
+        // Only add Google Scholar metadata tags to the canonical URL for the latest version
+        // See discussion: https://github.com/pkp/pkp-lib/issues/4870
```

Comment:
Looking at the discussion there, should this not also apply to OJS and OPS?

Reply:
Yes, it is applied everywhere...